docs/source/usage/build-public.rst

```diff
@@ -16,6 +16,10 @@ build-public
     The actions are performed in this order, meaning that *internal* properties will be taken into account when matching the JSON schema.
     If a component containing nested components is deleted, those nested components are deleted as well.
 
+    .. warning::
+        **Warning!** The ``metadata.component`` will not be removed even though the JSON schema applies to it. This is because no important information may be deleted.
```

Comment:
The description is wrong. The "even though" does not make sense, should be "even when", as the schema does not have to apply to it. Further, the reasoning should be "This is because the ``metadata.component`` is the component the BOM describes, therefore removing it, would make the SBOM ambiguous."